web/api.go

```diff
@@ -36,7 +41,7 @@ type ResultInput struct {
 	Stats         zoekt.Stats
 	Duration      time.Duration
 	FileMatches   []*FileMatch
-	SearchOptions string
+	SearchOptions zoekt.SearchOptions
```

Comment:
I don't see any updates to the templates, either this field was never read (so you don't need it) or there is a runtime bug when rendering the template. If go just magically converts this to a string, I wonder if the output has degraded. IE looks ugly / includes the defaults rather than what the user specified.

Reply:
Yes the field was never read in the templates from what I can see but I'm using it in a project I'm experimenting with. I'm fine with removing it though. WDYT?

Reply:
Yeah if it wasn't used then lets remove it :)